src/game.cpp

```diff
@@ -237,7 +237,7 @@ game::game() :
     events( *event_manager_ptr ),
     weather( WEATHER_CLEAR ),
     lightning_active( false ),
-    u_shared_ptr( &u, null_deleter{} ),
+    u_shared_ptr( &u, null_deleter {} ),
```

Comment:
You've made lots of unrelated whitespace changes like this one.  Please revert those.  They make the PR harder to review, and don't match the project style.

Reply:
it's not me, i haven't touched this line personally, it's the 'make astyle'.